Integrate-Kobiton.md

```diff
@@ -0,0 +1,156 @@
+    - [1. Executing the automation test](#1-executing-the-automation-test)
+    - [2. Fetching test session data using Kobiton REST API](#2-fetching-test-session-data-using-kobiton-rest-api)
+- [D. Feedback](#d-feedback)
+
+## A. Preparation
+### 1. Getting Kobiton Username and API key
+Kobiton Username and API key are required for authenticating with Kobiton REST API.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+### 2. Samples
+In this repository, we have already provided samples for executing automation test in Kobiton:
+- Script for executing automation test on Kobiton iOS devices.
+- Script for executing automation test on Kobiton Android devices.
+- CircleCI configuration file.
+
+Link to the sample: https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative/tree/master/sample
+
+In this guide, as we will be using the provided example, therefore, follow these steps below:
+
+1. Fork this repository https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative
+
+2. Synchronize the forked repository with CircleCI. If you don't know how to do it, refer to instruction at [CircleCI document](https://circleci.com/docs/2.0/getting-started/)
+
+3. Follow next sections to get desired capabilities and configure all necessary variables for running automation test.
+
+## B. Setup
+### 1. Setting Kobiton Username and API key
+
+In your project configuration in CircleCI, add these two environment variables :
+- KOBITON_USERNAME - Your Kobiton's username.
+- KOBITON_API_KEY - Your Kobiton's API Key.
+
+Your `Environment Variables` window should look like this
+
+![alt text](./assets/env-variables-after-add.png)
+
+### 2. Getting Kobiton device desired capabilities
+In order to interact with a specific device in Kobiton, its corresponding desired capabilities needs to be supplied.
+
+To get desired capabilities of a Kobiton device, follow these steps below :
+
+1. Go to [https://portal.kobiton.com/login](https://portal.kobiton.com/login) and login with your Kobiton account.
+2. Click **"Devices"** at the top of the window.
+
+![Devices button](./assets/devices-button.png)
+
+3. Hover over the device you want to run your test script on, click the gear button.
+
+![Automation Settings button](./assets/gear-button.png)
+
+4. In the popup window, you can choose the language of your script, application testing type, etc. In this example, we will be using NodeJS, therefore, change `Language` section to `NodeJS`. Then, change the value of `App Type` to `Hybrid/Native from Apps`. The desired capabilities and Kobiton server web driver configuration code will be generated and displayed in the right pane of the popup (marked **blue**).
+
+![Automation Settings](./assets/desired-caps.png)
```

Comment:
Replace this image where we only highlight the `desiredCaps` in blue.
Visit my repo for the image file.